examples/tst_pkg_huge.pks

```diff
@@ -305,6 +305,8 @@ create or replace package tst_pkg_huge is
     -- %test(Test number 100)
     procedure test100;
     
+$IF $$tst_slim $THEN
```

Comment:
well the point was to show/test that the get_source only get the correct CC compiled code..  so if you have packages under test that use conditional compilation it gets tested correctly still.. but sure.. it should probably be moved to test.. and not example.